src/space/objects/realization/loop-hole.js

```diff
@@ -0,0 +1,124 @@
+import Vector from "../../utils/vector";
+import EncounterObject from "../pattern/encounterObject";
+import Type from "../pattern/encounterObjectType";
+import MyMath from "../../utils/myMath";
+import AssetLoader from "../../utils/assetLoader";
+
+const LH_SPRITE_RADIUS = 20;
```

Comment:
I would also prefer using `LH_SPRITE_WIDTH` and `LH_SPRITE_HEIGHT` but it is not as crucial as in BlackHole.

Reply:
Can I use LH_SPRITE_SIZE, since those two values should be the same

Reply:
Same as before, it is ok provided no one will ever decide to have rectangular loopholes :)